### PR TITLE
AT_024.04.04 | Entity > Platform functions > Delete record

### PR DESCRIPTION
**Pull request by AnastasiiaPV**
AT_024.04.04 | Entity > Platform functions > Delete record
https://trello.com/c/zcSlz5Jf/1030-at0240404-entity-platform-functions-delete-record
RF | EntityPlatformFunctions1Test > general
https://trello.com/c/y4e36hRK/1056-rf-entityplatformfunctions1test-general

### CODE REVIEW

**Review by kololesya — APPROVED** [review]

**Review by Stan575 — CHANGES_REQUESTED** [review]

File: src/test/java/EntityPlatformFunctions1Test.java [hunk 1]
```diff
@@ -211,14 +218,37 @@ public void testSearchRecord() {
         }
     }
 
+    @Test(dependsOnMethods = "testSearchRecord")
+    public void testDeleteRecords() {
+        openEntityPlatformFunctions();
+        moveToClickOnElementAct(By.xpath(FIRST_MENU_IN_ROW));
+        waitToBeClickableClick(By.linkText(DELETE));
+        waitToBeClickableClick(By.xpath(SHOWING_1_TO_2_OF_2_ROWS));
```
[the remark this answers is not in the document]
Comment (Stan575) [reply src/test/java/EntityPlatformFunctions1Test.java:226]: see line 226, line 233 does essentially the same

[the remark this answers is not in the document]
Comment (AnastasiiaPV) [reply src/test/java/EntityPlatformFunctions1Test.java:226]: I came up with this Idea just to have additional time to wait till three lines will be changed by two lines.

[the remark this answers is not in the document]
Comment (AnastasiiaPV) [reply src/test/java/EntityPlatformFunctions1Test.java:226]: And this seems to me like good additional asserting.

[the remark this answers is not in the document]
Comment (Stan575) [reply src/test/java/EntityPlatformFunctions1Test.java:226]: got it, when you exploit some kinda workaround waiter it's good to make it easy to understand by just looking at the code
in this case it looks very suspicious for another person who doesn't know your intention

**Review by Stan575 — APPROVED** [review]
approved, think how to make that waiting part looking better

File: src/test/java/EntityPlatformFunctions1Test.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnastasiiaPV) [reply src/test/java/EntityPlatformFunctions1Test.java:226]: Thank you, Stan. Will try to do my best.

**Pull request merged**